### PR TITLE
First UnitTest (for ShopDistributor Method)

### PR DESCRIPTION
**Pull request by valentk777**
hey! I get illness, so possible do not have a possibility to take part in our lectures today. Also, I do not have time to finish all unit test as I wanted. I will do it as fast as possible. 

Now I prepared 4 unit tests wit XUnit for method ShopDistrobutor. Feel free to give me feedback and advice for future testing. 

--
regards,
Valentinas

### CODE REVIEW

**Comment by valentk777** [comment]
does anyone can have a look watch wrong with integration to github? as everyone sow, everything work in our local repositories.

File: GREEDY/View/MainScreen.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ private void InserFile_Button_Click(object sender, EventArgs e)
             InserFile_Button.Enabled = true;
         }
 
-        private void itemList_CellValueChanged(object sender, DataGridViewCellEventArgs e)
+        private void ItemList_CellValueChanged(object sender, DataGridViewCellEventArgs e)
```
Comment (MariusKurbakovas) [inline GREEDY/View/MainScreen.cs:41]: Thanks for this change, missed it

**Review by MariusKurbakovas — APPROVED** [review]
I think it's all good, I'll start writing tests like these for my classes too

**Review by MariusKurbakovas — COMMENTED** [review]
About the continious integration, there are few configuration files missing, Aidas will add them

**Review by MariusKurbakovas — COMMENTED** [review]
Added continious integration (working)

**Pull request merged**